DUI3/Speckle.Connectors.DUI/Models/Card/ReceiverModelCard.cs

```diff
@@ -12,13 +12,15 @@ public class ReceiverModelCard : ModelCard
   public bool HasDismissedUpdateWarning { get; set; }
   public List<string>? BakedObjectIds { get; set; }
 
-  public ReceiveInfo GetReceiveInfo() =>
+  public ReceiveInfo GetReceiveInfo(string sourceApplication) =>
     new(
       AccountId.NotNull(),
       new Uri(ServerUrl.NotNull()),
       ProjectId.NotNull(),
       ProjectName.NotNull(),
+      ModelId.NotNull(),
       ModelName.NotNull(),
-      SelectedVersionId.NotNull()
+      SelectedVersionId.NotNull(),
+      sourceApplication
```

Comment:
I'm assuming you already looked into injecting the source application somehow but this was the easiest way through?

It seems a bit odd to have to pass in the hostApp to get the "receive info"

Reply:
It is, I agree. but this is what we are doing with the SendInfo.

I think we should proably come up with a different design. Our settings (`RhinoSettings`, `AutoCADSettings`, etc..) are all pretty different looking, which is a shame. I might ping @BovineOx see if we should create a ticket

Reply:
https://spockle.atlassian.net/browse/DUI3-484